packages/web-react/src/components/Card/README.md

```diff
@@ -0,0 +1,150 @@
+# Card
+
+Basic usage:
+
+```jsx
+<Card>…</Card>
+```
+
+Advanced example:
+
+```jsx
+<Card elementType="article" direction="horizontal" isBoxed>
+  …
+</Card>
+```
+
+Card composition example:
```

Comment:
After updating according to the `web` README, this will become the "Full Example" section.